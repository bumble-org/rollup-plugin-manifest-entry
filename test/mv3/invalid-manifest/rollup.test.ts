import { rollup, RollupOptions } from 'rollup'
import config from './rollup.config'

process.chdir(__dirname)

test('errors if the manifest is invalid', async () => {
  expect.assertions(1)
  await expect(
    rollup(config as RollupOptions),
  ).rejects.toMatchSnapshot('error')
})
