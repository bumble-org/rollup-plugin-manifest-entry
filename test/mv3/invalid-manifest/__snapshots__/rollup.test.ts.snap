// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`errors if the manifest is invalid: error 1`] = `
[Error: There were problems with the extension manifest.
- ["service_worker.js"] at "manifest.background.service_worker" must be string]
`;
